### PR TITLE
add serializer

### PR DESCRIPTION
**Pull request by Leptopoda**

### CODE REVIEW

File: packages/dynamite/lib/src/openapi_builder.dart [hunk 1]
```diff
@@ -1016,6 +1019,12 @@ class OpenAPIBuilder implements Builder {
 
       if (state.registeredJsonObjects.isNotEmpty) {
         output.addAll([
+          '@SerializersFor(const [',
+          for (final name in state.registeredJsonObjects) ...[
+            '$name,',
+          ],
+          '])',
+          r'final Serializers serializers = (_$serializers.toBuilder()..addPlugin(StandardJsonPlugin())).build();',
```
Comment (provokateurin) [inline packages/dynamite/lib/src/openapi_builder.dart:1027]: Should the `_serializers` below removed after this change?

**Review by provokateurin — CHANGES_REQUESTED** [review]

File: packages/dynamite/lib/src/openapi_builder.dart (hunk 1 above)
Comment (Leptopoda) [reply packages/dynamite/lib/src/openapi_builder.dart:1027]: I have it in a separate commit as theoretically the normal objects aren't yet migrated to it.

[the remark this answers is not in the document]
Comment (Leptopoda) [reply packages/dynamite/lib/src/openapi_builder.dart:1027]: they both serialize and deserialize

File: packages/dynamite/lib/src/openapi_builder.dart [hunk 2]
```diff
@@ -65,6 +65,9 @@ class OpenAPIBuilder implements Builder {
         '',
         "import 'package:built_collection/built_collection.dart';",
         "import 'package:built_value/built_value.dart';",
+        "import 'package:built_value/json_object.dart';",
+        "import 'package:built_value/serializer.dart';",
+        "import 'package:built_value/standard_json_plugin.dart';",
```
[the remark this answers is not in the document]
Comment (Leptopoda) [reply packages/dynamite/lib/src/openapi_builder.dart:70]: Nope for some reason they aren't.

In theory we don't need `"import 'package:built_value/json_object.dart';",` but it'll come in the next PR so I could take it out again.

File: packages/dynamite/lib/src/openapi_builder.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (provokateurin) [reply packages/dynamite/lib/src/openapi_builder.dart:1027]: Ah, didn't know that :+1:

Comment (provokateurin) [reply packages/dynamite/lib/src/openapi_builder.dart:1027]: Could you already do the migration in this PR or is it too big?

File: packages/dynamite/lib/src/openapi_builder.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (provokateurin) [reply packages/dynamite/lib/src/openapi_builder.dart:70]: Ok whatever.

File: packages/dynamite/lib/src/openapi_builder.dart (hunk 1 above)
Comment (Leptopoda) [reply packages/dynamite/lib/src/openapi_builder.dart:1027]: I can sqash them if you want github.com/provokateurin/nextcloud-neon/pull/194/commits/b1c4273981ea9fc8e168ac78f04afeb279de9239

Comment (provokateurin) [reply packages/dynamite/lib/src/openapi_builder.dart:1027]: Yes please, the commit looks good.

**Review by provokateurin — APPROVED** [review]

**Pull request merged**